cogs5e/models/homebrew/bestiary.py

```diff
@@ -415,7 +415,7 @@ def _monster_factory_bestiary_builder(data, bestiary_name):
         la_per_round=data["la_per_round"],
         attacks=attacks,
         proper=data["proper"],
-        image_url="",
+        image_url=data["image_url"],
```

Comment:
@VeryGreatFrog If there's no image, does your export give a blank value for this, or are we going to have KeyErrors ?

Reply:
It will be an empty string: https://github.com/Bestiary-Builder/Bestiary-Builder/blob/main/backend/logic/bestiaries.ts#L556

Reply:
Excellent, let's get this merged into nightly for testing then :)